README.md

```diff
@@ -45,20 +45,41 @@ make package
 to build the REST API package.
 
 ## Configuring the service
+The REST API provides an executable for each individual API endpoint. These endpoints may be grouped behind a reverse proxy in order to provide a single port for access.
 
-The REST API provides an executable for each individual API endpoint.  These endpoints may be grouped behind a reverse proxy in order to provide a single port for access.
-
-The services rely on a configuration file in `/etc/irods/` which dictates which ports are used for each service.  Two template files are placed there by the package:
+The services rely on a configuration file in `/etc/irods` which dictates which ports are used for each service. Two template files are placed there by the package:
 ```
 /etc/irods/irods_client_rest_cpp.json.template
-/etc/irods/irods-client-rest-cpp-reverse-proxy.conf.template
+/etc/irods/irods_client_rest_cpp_reverse_proxy.conf.template
 ```
-`/etc/irods/irods_client_rest_cpp.json.template` should be copied to `/etc/irods/irods_client_rest_cpp.json` and modified if different ports are desired.  The service can then be restarted with `service irods_client_rest_cpp restart`, or however your specific platform manages services.
+`/etc/irods/irods_client_rest_cpp.json.template` should be copied to `/etc/irods/irods_client_rest_cpp.json` and modified if different ports are desired. The service can then be restarted with `sudo systemctl restart irods_client_rest_cpp`, or however your specific platform manages services.
```

Comment:
let's provide a bash example, like below in the logging section

Reply:
done